src/Cli/dotnet/commands/dotnet-workload/install/IWorkloadManifestUpdater.cs

```diff
@@ -11,6 +11,8 @@ internal interface IWorkloadManifestUpdater
     {
         Task UpdateAdvertisingManifestsAsync(SdkFeatureBand featureBand, bool includePreviews);
 
+        IEnumerable<string> GetManifestsUrls(SdkFeatureBand featureBand, bool includePreviews);
```

Comment:
Consider renaming this to `GetManifestPackageUrls`, I think that would be more clear.

How would this work with the standalone IAL?  Would this be returning the Urls to the NuGet packages that contain the MSIs?  But IIRC this interface isn't part of the IAL.

Reply:
I'm going to back this out for now since I'll be addressing manifest caching in a separate PR.